nym-api/src/node_describe_cache/mod.rs

```diff
@@ -248,13 +304,13 @@ impl CacheItemProvider for NodeDescriptionProvider {
 #[allow(dead_code)]
 pub(crate) fn new_refresher(
     config: &config::TopologyCacher,
-    contract_cache: NymContractCache,
+    node_status_cache: NodeStatusCache,
```

Comment:
any reason we switched from using raw contract data for figuring out what to query in favour of the fat node status cache with all the inclusion probabilities, performance, families, etc?

Reply:
> any reason we switched from using raw contract data for figuring out what to query in favour of the fat node status cache with all the inclusion probabilities, performance, families, etc?

Yes, we need the annotated bonds to get the layer info for mixnodes, so that we can reuse the NodeRole thing we already have